calblueprint/siren
# Doc upload new

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
Finished @cnnmon 's work on doc upload

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [X] Take Picture 
 - [X] Upload Picture to Firebase Storage
 - [ ] Update Document object in Firebase Firestore (pending Noah document PR)

## How can the reviewer test your code? 👩‍💻
log in
go to forms screen
take picture
click keep scan
verify here: https://console.firebase.google.com/u/0/project/siren-6099f/storage/siren-6099f.appspot.com/files

## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation_ 


## Resources 📔
_link online resources and related PRs_ 


🧜‍♀️ cc: @shannonbonet @phoebeli23
